api/app/models/land_change/carbon_flux.py

```diff
@@ -0,0 +1,91 @@
+from typing import Annotated, List, Optional, Union
+
+from app.models.common.analysis import AnalysisStatus, AnalyticsIn
+from app.models.common.areas_of_interest import (
+    AdminAreaOfInterest,
+    CustomAreaOfInterest,
+    IndigenousAreaOfInterest,
+    KeyBiodiversityAreaOfInterest,
+    ProtectedAreaOfInterest,
+)
+from app.models.common.base import Response, StrictBaseModel
+from pydantic import Field
+
+AoiUnion = Union[
+    AdminAreaOfInterest,
+    KeyBiodiversityAreaOfInterest,
+    ProtectedAreaOfInterest,
+    IndigenousAreaOfInterest,
+    CustomAreaOfInterest,
+]
+
+DATE_REGEX = r"^\d{4}$"
+
+
+class CarbonFluxAnalyticsIn(AnalyticsIn):
+    aoi: Annotated[AoiUnion, Field(discriminator="type")] = Field(
```

Comment:
I think you do need to support canopy cover option here, for thresholds 30, 50 and 75